dbt/parser.py

```diff
@@ -399,7 +396,7 @@ def parse_schema_test(test_base, model_name, test_config, test_type,
 
     raw_sql = "{{{{ {macro}(model=ref('{model}'), {kwargs}) }}}}".format(**{
         'model': model_name,
-        'macro': "test_{}".format(test_type),
+        'macro': "dbt.test_{}".format(test_type),
```

Comment:
this is ok for now, but i think ideally we wouldn't namespace this. Idea being that end users could 1) define their own tests (not in the dbt namespace) or 2) override existing tests